scripts/submitSparkPi.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#command should be run like so:
+#  examples/submitSparkPi.sh armada://localhost:30002 ~/incoming/spark
+
+ARMADA_MASTER=$1
+SPARK_ROOT=$2
+ARMADA_SPARK_ROOT=`pwd`
+ARMADA_QUEUE=test
+# Ensure our "test" queue exists on Armada
+armadactl get queues | grep -q -w $ARMADA_QUEUE
```

Comment:
Firstly, shouldn't we use this command?
```suggestion
armadactl get queue $ARMADA_QUEUE
```

Reply:
this commit  suggestion was superseded by your following one, so I took that one instead.